### ISSUE TITLE
`no-negated-condition` rule wrongly marks and autofix case when `not` have multiple parameters

### ISSUE THREAD

**Issue by godric3** [ember-template-lint/ember-template-lint#2647]
```hbs
{{#if (not c1 c2)}}{{/if}}
```
is autofixed into
```hbs
{{#unlesss c1}}{{/unless}}
```
completely skipping all parameters after first one.
I think autofixing that into simple thing is not possible because it would require `NOR` helper, so this should at least not be marked as violation.

Didn't tested that on real codebase but only in rule tests.
